src/licensedcode/data/licenses/kazlib.LICENSE

```diff
@@ -0,0 +1,20 @@
+---
+key: kazlib
+short_name: Kazlib
+name: Kazlib License
+category: Permissive
+owner: Kaz Kylheku
+spdx_license_key: Kazlib
+other_urls:
+    - http://git.savannah.gnu.org/cgit/kazlib.git/tree/except.c?id=0062df360c2d17d57f6af19b0e444c51feb99036
+---
+
+Free Software License: 
```

Comment:
https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/rules/other-permissive_92.RULE needs to be requalified